modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,166 @@ The properties for broker-wide throughput quota balancing are configured at the
-  {
-    "group_name": "group_1",
-    "clients_prefix": "producer_group_alone_producer",
-    "quota": 10240
-  },
-  {
-    "group_name": "group_2",
-    "clients_prefix": "producer_group_multiple",
-    "quota": 20480
-  },
++
+[,bash,role=no-copy]
+----
+"kafka_client_group_byte_rate_quota": [ 
+  { 
+    "group_name": "group_1", 
+    "clients_prefix": "producer_group_alone_producer", 
+    "quota": 10240 
+  }, 
+  { "group_name": "group_2", 
+    "clients_prefix": "producer_group_multiple", 
+    "quota": 20480 
+  }, 
 ]
 ----
+
+. Each client quota cluster property corresponds to a quota type in Kafka. Check the corresponding `rpk` arguments to use when setting the new quota values:
++
+|===
+| Cluster configuration property | `rpk cluster quotas` arguments
+
+| xref:reference:properties/cluster-properties.adoc#target_quota_byte_rate`[`target_quota_byte_rate`]                      
+| `--default client-id --add producer_byte_rate=<producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#target_fetch_quota_byte_rate[`target_fetch_quota_byte_rate`]                
+| `--default client-id --add consumer_byte_rate=<consumer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_admin_topic_api_rate[`kafka_admin_topic_api_rate`]                  
+| `--default client-id --add controller_mutation_rate=<controller-mutation-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_byte_rate_quota[`kafka_client_group_byte_rate_quota`]          
+| `--name client-id-prefix=<prefix> --add producer_byte_rate=<group-producer-byte-rate-value>`
+
+| xref:reference:properties/cluster-properties.adoc#kafka_client_group_fetch_byte_rate_quota[`kafka_client_group_fetch_byte_rate_quota`]    
+| `--name client-id-prefix=<prefix> --add consumer_byte_rate=<group-consumer-byte-rate-value>`
+
+|===
++
+The client throughput quotas set through the Kafka API apply per broker, so you need to convert the cluster configuration values that were applied on a per-shard (logical CPU core) basis. For example, if you set `target_fetch_quota_byte_rate` to 100MBps/shard, and you run Redpanda on 16-core brokers, you can set the new consumer_byte_rate quota to 100 * 16 = 1600MBps.
+
+. Use `rpk cluster quotas alter` to set the corresponding client throughput quotas based on the Kafka API:
++
+[,bash]
+----
+rpk cluster quotas alter --name client-id-prefix=producer_group_alone_producer --add producer_byte_rate=<group-1-producer-byte-rate-value> 
+rpk cluster quotas alter --name client-id-prefix=producer_group_multiple --add producer_byte_rate=<group-2-producer-byte-rate-value> 
+----
++
+Replace the placeholder values with the new quota values, accounting for the conversion to per-broker limits, for example 10240 * broker core count = new quota.
+
+. Use `rpk cluster config` to remove the configuration-based quotas:
++
+[,bash]
+----
+rpk cluster config kafka_client_group_byte_rate_quota=
```

Comment:
should there be a placeholder here for to the user to replace with their own value (inline edit of the example)?

Reply:
@Feediver1 This syntax "clears out" the config, but actually @r-vasquez could you confirm if the explanation is correct? Based on the existing reference doc I'm not sure that the command actually removes the quota (it says "reset to default").

Reply:
Yes, the default is an empty list and should be fine.

However I think it's missing the `set`:
```suggestion
rpk cluster config set kafka_client_group_byte_rate_quota=
```